templates/table.html

```diff
@@ -51,12 +42,45 @@ <h3>{% if filtered_table_rows or filtered_table_rows == 0 %}{{ "{:,}".format(fil
             </div><input type="text" name="_filter_value_{{ loop.index }}" class="filter-value" value="{{ value }}">
         </div>
     {% endfor %}
-    <div class="filter-row">
+    <div class="dbsearch filter-row">
         <div class="select-wrapper">
+            Filter: 
             <select name="_filter_column">
                 <option value="">- column -</option>
                 {% for column in filter_columns %}
+                    {% if column == "first_name" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "last_name" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "organization" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "agency" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "class_code" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "ytd_gross_earnings" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "annual_salary" %}
                       <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "regular_earnings" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "overtime_earnings" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "other_earnings" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
+                    {% if column == "term_date" %}
+                      <option>{{ column }}</option>
+                    {% endif %}
```

Comment:
Isn't this all the columns? Do we need this?

Reply:
it's not (i doubt users will need to search by middle initial or suffix, etc.) i did make this a little cleaner, though